src/openforms/prefill/api/views.py

```diff
@@ -6,6 +6,15 @@
 from rest_framework.views import APIView
 
 from openforms.api.views import ListMixin
+from openforms.registrations.contrib.objects_api.api.serializers import (
+    ObjectsAPIGroupInputSerializer,
+)
```

Comment:
The open forms development guidelines forbid importing stuff from one django app in another django app (with the exception of the `models` and `service` modules). This principle is even stronger when considering Open Forms modules (registrations, prefill, appointments...) and plugins within modules (objects_api, stuf-bg, ...).

The reason for that is that a plugin (and django app) must be as self-contained as possible - the documentation on [RTD](https://open-forms.readthedocs.io/en/stable/developers/backend/index.html) tries to explain this.

In this case, it's better to duplicate the serializer implementation. Yes, they currently do similar/the same things, but that doesn't make them the same. The prefill and registration plugins must be evolvable independently from each other.

If you really want to share this, the serializer should be moved to `openforms.contrib.objects_api` instead, so that it can be re-used by the registrations and prefill plugins, and the hierarchy is clear.